docs/api.rst

```diff
@@ -13695,12 +13697,44 @@ Managing calendar reminders
       Content-Type: application/json
 
       {
-        "result": true,
-        "message": ""
+        "message": "",
+        "result": {
+          "eth_on_gas": "0.21760082633497774",
+          "gnosis_max_payments_by_currency": {
+              "AED": "36.0",
+              "EUR": "120.17",
+              "THB": "13615.25",
+              "USD": "85.0"
+          },
+          "top_days_by_number_of_transactions": [
```

Comment:
what is this? It's not explained in the docs well and not sure I understand.

Reply:
it is like the heatmap on github. Returns the 10 days when you transacted the most